src/seo.ts

```diff
@@ -215,8 +215,15 @@ export class Seo extends BasePlugin<Record<string, never>> {
         captionData.push(cue);
       }
     });
-    if (chapterData.length && this.cuesSource === CueSourceNames.None) {
+    if (chapterData.length && this.cuesSource !== CueSourceNames.Unisphere) {
```

Comment:
not a fan of multiple nested ifs
in this case
looks like you can make it a bit more readable by separating the ifs

if (chapterData.length && this.cuesSource !== CueSourceNames.Unisphere) {
  this.chaptersData = Seo.extractRelevantChaptersData(chapterData);
}

if (this.cuesSource === CueSourceNames.None) {
 ...
}